ds3-sdk/src/main/java/com/spectralogic/ds3client/serializer/XmlOutput.java

```diff
@@ -37,6 +44,34 @@
         mapper = new XmlMapper(module);
         final SimpleFilterProvider filterProvider = new SimpleFilterProvider().setFailOnUnknownId(false);
         mapper.setFilters(filterProvider);
+        if (isProductionBuild()) {
+            mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        }
+    }
+
+    protected static boolean isProductionBuild() {
+        final Properties props = new Properties();
+        final InputStream input = XmlOutput.class.getClassLoader().getResourceAsStream("/config.properties");
+        if (input == null) {
+            LOG.error("Could not find property file.");
+        }
+        else {
+            try {
+                props.load(input);
+                final String productionBuild = (String) props.get("productionBuild");
+                if (productionBuild.equals("true")) {
+                    return true;
+                }
+                else {
+                    LOG.error("Unknown productionBuild value[" + productionBuild + "].  Defaulting to fail for unknown XML elements.");
+                }
+            } catch (final IOException e) {
+                LOG.error("Failed to load property file.");
```

Comment:
Add the exception to the end of the log message so that it is printed out.  You should be able to just add it by doing `LOG.error("Failed to load property file.", e);`